packages/postgres/src/default-pool.ts

```diff
@@ -165,13 +165,6 @@ export const callOptionalRow = defaultPool.callOptionalRow.bind(defaultPool);
  * read results in batches, which is useful for large result sets.
  */
 export const queryCursorWithClient = defaultPool.queryCursorWithClient.bind(defaultPool);
-/**
- * Returns an {@link CursorIterator} that can be used to iterate over the
- * results of the query in batches, which is useful for large result sets.
- *
- * @deprecated Use {@link queryValidatedCursor} instead.
- */
-export const queryCursor = defaultPool.queryCursor.bind(defaultPool);
```

Comment:
Nit: consider renaming `queryValidatedCursor` to `queryCursor`. The "validated" distinction doesn't matter anymore as there's no non-validated function.

Reply:
Done in [57f42cf](https://github.com/PrairieLearn/PrairieLearn/pull/12402/commits/57f42cfdb82097681a5276fb37e5992b693b8e22).